js/bibox.js

```diff
@@ -662,7 +662,8 @@ module.exports = class bibox extends Exchange {
             const id = this.safeString (currency, 'symbol');
             const name = currency['name']; // contains hieroglyphs causing python ASCII bug
             const code = this.safeCurrencyCode (id);
-            const precision = this.parseNumber ('0.00000001');
+            const backupPrecision = this.numberToString (this.precisionFromString (this.safeString (currency, 'total_amount')));
```

Comment:
We should not really use precisionFromString on arbitrary numbers... Can you elaborate a bit on why would we take a value of `4776930644` to figure out the precision here ?

Reply:
after this PR i've found out bibox has updated API version, so i think we no longer need to update this api at all. closing this pr